src/main/java/com/liefery/android/stop_badge/StopBadgeFactory.java

```diff
@@ -0,0 +1,37 @@
+package com.liefery.android.stop_badge;
+
+import android.annotation.SuppressLint;
+import android.graphics.*;
+import android.util.LruCache;
+
+@SuppressLint( "NewApi" )
+public class StopBadgeFactory {
```

Comment:
I use a Singleton to be able to access the same cache from everywhere. Making the cache simply static wouldn't help us since every instance of `StopBadgeFactory` will have it's own cache, or am I seeing something wrong?